tcms/testruns/templates/testruns/get.html

```diff
@@ -16,6 +16,7 @@ <h1 class="col-md-12" style="margin-top: 0">
             data-pk="{{ object.pk }}"
             data-perm-remove-tag="{{ perms.testruns.delete_testruntag }}"
             data-trans-no-executions-selected="{% trans 'No executions selected! Please select at least one execution!'%}"
+            data-trans-enter-assignee-name-or-email="{% trans 'Enter username or email of the new assignee!'%}"
```

Comment:
The supported values are int (for PK), str for username/email so the message can be adjusted a bit.